ansible_collections/arista/cvp/molecule/cv_facts_v3/test_cv_facts_v3.yml

```diff
@@ -0,0 +1,244 @@
+---
+- name: Test cv_facts_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    CVP_CONFIGLETS:
+      configlet1: 'alias a{{ 999 | random }} show lldp'
+    CVP_DEVICES_MOVE_DEVICE:
+      - fqdn: leaf3
+        parentContainerName: "ATD_SPINES"
```

Comment:
can you create a container and then move the device. This container is not available on ATD by default

Reply:
Done!